tailwind.config.ts

```diff
@@ -117,11 +117,22 @@ export default {
       'ibm-plex-mono': ['var(--font-ibm-plex-mono)'],
     },
     extend: {
+      screens: {
+        xs: { max: '639px' },
+      },
       backgroundImage: {
         'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
+        'gradient-subtle':
+          // @todo: replace colors for proper tailwind vars (?)
```

Comment:
we can leave as it it's correct. what do you mean by tailwind var ? `--tw_XXX` ?

Reply:
I have updated with [tailwind theme function](https://tailwindcss.com/docs/functions-and-directives#theme)